### PR TITLE
gapc on Windows

### PR DESCRIPTION
**Pull request by sgtkellox**

### CODE REVIEW

File: documentation/gapc on windows.txt [hunk 1]
```diff
@@ -0,0 +1,23 @@
+How to compile gapc on Windows 10:
+
+1) Install Linux Sub-System
+	1.1) go Start -> Settings -> For Developers : Click Developer-Mode
+	1.2) Open Windows Power Shell then type: Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+	1.3) restart computer
+	1.4) Download Ubuntu app from Windows-Strore
```
Comment (sjanssen2) [inline documentation/gapc on windows.txt:7]: typo: St_r_ore

Comment (sjanssen2) [reply documentation/gapc on windows.txt:7]: is it possible to give a URL for that? Which Ubuntu version is currently used? 18.04?

File: documentation/gapc on windows.txt [hunk 2]
```diff
@@ -0,0 +1,23 @@
+How to compile gapc on Windows 10:
+
+1) Install Linux Sub-System
+	1.1) go Start -> Settings -> For Developers : Click Developer-Mode
+	1.2) Open Windows Power Shell then type: Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+	1.3) restart computer
+	1.4) Download Ubuntu app from Windows-Strore
+	1.5) Initialize installation by setting Username and Password
+2) Prepare compiling and installation of gapc by installing requiered sources: 
+	2.1) Open Ubuntu app
```
Comment (sjanssen2) [inline documentation/gapc on windows.txt:10]: you here might want to point to the .travis file in this repo. That should instruct on which packages to install

**Comment by sjanssen2** [comment]
@kmaibach, would you please also review this PR of Felix?

File: documentation/gapc on windows.txt [hunk 3]
```diff
@@ -0,0 +1,23 @@
+How to compile gapc on Windows 10:
+
+1) Install Linux Sub-System
+	1.1) go Start -> Settings -> For Developers : Click Developer-Mode
+	1.2) Open Windows Power Shell then type: Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+	1.3) restart computer
+	1.4) Download Ubuntu app from Windows-Strore
+	1.5) Initialize installation by setting Username and Password
+2) Prepare compiling and installation of gapc by installing requiered sources:
```
Comment (kmaibach) [inline documentation/gapc on windows.txt:9]: typo: requi_e_red

File: documentation/gapc on windows.txt [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+	1.3) restart computer
+	1.4) Download Ubuntu app from Windows-Strore
+	1.5) Initialize installation by setting Username and Password
+2) Prepare compiling and installation of gapc by installing requiered sources: 
+	2.1) Open Ubuntu app
+	2.2) install compiler: sudo apt update
+			       sudo apt install gcc
+			       sudo apt install g++ (for some reason u need both)	
+	2.3) install flex    : sudo apt-get install flex
+	2.4) install bison   : DO NOT :sudo apt-get install bison !!
```
Comment (kmaibach) [inline documentation/gapc on windows.txt:15]: typo: move the second ':' (DO NOT: sudo...)

File: documentation/gapc on windows.txt [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+	1.4) Download Ubuntu app from Windows-Strore
+	1.5) Initialize installation by setting Username and Password
+2) Prepare compiling and installation of gapc by installing requiered sources: 
+	2.1) Open Ubuntu app
+	2.2) install compiler: sudo apt update
+			       sudo apt install gcc
+			       sudo apt install g++ (for some reason u need both)	
+	2.3) install flex    : sudo apt-get install flex
+	2.4) install bison   : DO NOT :sudo apt-get install bison !! 
+			       INSTEAD go on the bison website and istall the latest version
```
Comment (kmaibach) [inline documentation/gapc on windows.txt:16]: typo: i(n)stall
maybe provide a URL to the bison website

File: documentation/gapc on windows.txt [hunk 6]
```diff
@@ -0,0 +1,23 @@
+How to compile gapc on Windows 10:
+
+1) Install Linux Sub-System
+	1.1) go Start -> Settings -> For Developers : Click Developer-Mode
+	1.2) Open Windows Power Shell then type: Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
```
Comment (kmaibach) [inline documentation/gapc on windows.txt:5]: You should try to be consistent with your capitalization, eg. 1.1) *g*_o vs. 1.2) *O*_pen

File: documentation/gapc on windows.txt [hunk 7]
```diff
@@ -0,0 +1,23 @@
+How to compile gapc on Windows 10:
+
+1) Install Linux Sub-System
```
Comment (kmaibach) [inline documentation/gapc on windows.txt:3]: there's a ':' missing

File: documentation/gapc on windows.txt [hunk 8]
```diff
@@ -0,0 +1,23 @@
...
+	1.1) go Start -> Settings -> For Developers : Click Developer-Mode
+	1.2) Open Windows Power Shell then type: Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+	1.3) restart computer
+	1.4) Download Ubuntu app from Windows-Strore
+	1.5) Initialize installation by setting Username and Password
+2) Prepare compiling and installation of gapc by installing requiered sources: 
+	2.1) Open Ubuntu app
+	2.2) install compiler: sudo apt update
+			       sudo apt install gcc
+			       sudo apt install g++ (for some reason u need both)
```
Comment (kmaibach) [inline documentation/gapc on windows.txt:13]: I'd leave out the 'for some reason' and write out the 'you'

File: documentation/gapc on windows.txt (hunk 1 above)
Comment (sgtkellox) [reply documentation/gapc on windows.txt:7]: I made sure which Ubuntu version to use but the "Windows Store" is an app and I dont know if its possible to link a spezific source.  I think it should be clear what to do. Its like using Appstore or Playstore on your phone.

File: documentation/gapc on windows.txt [hunk 9]
```diff
@@ -0,0 +1,26 @@
...
+1) install Linux Sub-System:
+	1.1) go Start -> Settings -> For Developers : Click Developer-Mode
+	1.2) open Windows Power Shell then type: Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux
+	1.3) restart computer
+	1.4) download Ubuntu app (Ubuntu 18.04 LTS) from Windows-Store
+	1.5) initialize installation by setting Username and Password
+2) prepare compiling and installation of gapc by installing required sources: 
+	2.1) open Ubuntu app
+	2.2) install compiler: sudo apt update
+			       sudo apt install gcc
```
Comment (sjanssen2) [inline documentation/gapc on windows.txt:12]: why is it here `sudo apt install` and further down `sudo apt-get install`?

File: documentation/gapc on windows.txt [hunk 10]
```diff
@@ -0,0 +1,26 @@
...
+			       sudo apt install g++ (you need to install both)	
+	2.3) install flex    : sudo apt-get install flex
+	2.4) install bison   : DO NOT: sudo apt-get install bison !! 
+			       INSTEAD visit the bison website and install the latest version
+			       http://mirrors.ibiblio.org/gnu/ftp/gnu/bison/	
+	2.5) install boost   : sudo apt-get install libboost-all-dev (takes some time)
+	2.6) install GSL     : sudo apt-get install libgsl-dev
+	
+	--see also https://github.com/jlab/gapc/blob/master/.travis.yml
```
Comment (sjanssen2) [inline documentation/gapc on windows.txt:21]: can we add something like "An automated build script to compile Bellman's GAP is used to execute continuous integration tests and might be helpful as further reference: https://github.com/jlab/gapc/blob/master/.travis.yml"

File: documentation/gapc on windows.txt [hunk 11]
```diff
@@ -0,0 +1,26 @@
...
+	1.5) initialize installation by setting Username and Password
+2) prepare compiling and installation of gapc by installing required sources: 
+	2.1) open Ubuntu app
+	2.2) install compiler: sudo apt update
+			       sudo apt install gcc
+			       sudo apt install g++ (you need to install both)	
+	2.3) install flex    : sudo apt-get install flex
+	2.4) install bison   : DO NOT: sudo apt-get install bison !! 
+			       INSTEAD visit the bison website and install the latest version
+			       http://mirrors.ibiblio.org/gnu/ftp/gnu/bison/
```
Comment (sjanssen2) [inline documentation/gapc on windows.txt:17]: could you note which version you are currently using? If something breakes in the future, people can start to read the changelog from this version up to the current state

**Comment by sjanssen2** [comment]
no testing for windows, thus not sure if description is correct

**Pull request closed without merging**